code/modules/power/supermatter/supermatter.dm

```diff
@@ -776,10 +734,13 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			if(prob(10))
 				T.hotspot_expose(max(((100-support_integrity)*2)+FIRE_MINIMUM_TEMPERATURE_TO_EXIST,T.return_air().return_temperature()), 100)
 			if(prob(10+round(support_integrity/10,1)))
+				pulsewave()
 				var/ballcount = round(10-(support_integrity/10), 1) // Cause more radballs to be spawned
 				for(var/i = 1 to ballcount)
 					fire_nuclear_particle()
 		if(support_integrity<10)
+			powerloss_inhibitor = 0.01 //ensure big explosion
+			surging = 10000
```

Comment:
Should this be a define instead? And why does it need to be this flat number?

Reply:
nah, the `surging` will be added to `power` to increase the explosion strength